test/framework/options.py

```diff
@@ -2248,7 +2270,11 @@ def test_include_module_naming_schemes(self):
         args = [
             '--avail-module-naming-schemes',
         ]
-        logtxt, _ = run_cmd("cd %s; eb %s" % (self.test_prefix, ' '.join(args)), simple=False)
+
+        env_cmd = "export PATH='%s'; export PYTHONPATH='%s'; " % (path, pythonpath)
```

Comment:
Don't really need the "; export" part here you can export both with the same export